core/src/main/java/org/apache/brooklyn/core/effector/MethodEffector.java

```diff
@@ -133,6 +133,10 @@ public MethodEffector(Method method) {
         this(new AnnotationsOnMethod(method.getDeclaringClass(), method), null);
     }
 
+    /**
+     * @deprecated since 0.11.0; explicit groovy utilities/support will be deleted.
+     */
+    @Deprecated
```

Comment:
My rule-of-thumb... if one might get at the code via a yaml blueprint (so never seeing a deprecation warning in your IDE/compiler), then we should include a log.warn. But if we can assume that the user will get compiler warnings, then that is sufficient.

I believe that this will only get called directly in code, so one should see a compiler warning.